ui/src/pages/home/apps-section/app-filters/app-filters.tsx

```diff
@@ -209,6 +231,28 @@ export const AppFilters = ({
                 ))}
               </Box>
               <Divider sx={{ mt: '24px', mb: '16px' }} />
+              <FormLabel
+                id="server-statuses-label"
+                sx={{
+                  pb: '16px',
+                  fontSize: '14px',
+                  fontWeight: 600,
+                }}
+              >
+                Server Statuses
```

Comment:
This should be `Server Status` according to the design

Reply:
Updated